projects/plugins/jetpack/extensions/shared/components/product-management-controls/inspector-control.js

```diff
@@ -0,0 +1,126 @@
+	const { saveProduct } = useDispatch( membershipProductsStore );
+
+	const [ apiState, setApiState ] = useState( API_STATE_NOT_REQUESTING );
+	const [ title, setTitle ] = useState( __( 'Monthly Subscription', 'jetpack' ) );
+	const [ currency, setCurrency ] = useState( 'USD' );
+	const [ price, setPrice ] = useState( 5 );
+	const [ interval, setInterval ] = useState( '1 month' );
+
+	const intervalOptions = [
+		{ label: __( 'Month', 'jetpack' ), value: '1 month' },
+		{ label: __( 'Year', 'jetpack' ), value: '1 year' },
+	];
+	if ( allowCreateOneTimeInterval ) {
+		intervalOptions.push( { label: __( 'One-Time Payment', 'jetpack' ), value: 'one-time' } );
+	}
+
+	const handleSubmit = event => {
+		event.preventDefault();
+		setApiState( API_STATE_REQUESTING );
+		saveProduct( { title, currency, price, interval }, setSelectedProductId, success => {
+			setApiState( API_STATE_NOT_REQUESTING );
+			if ( success ) {
+				setPrice( 5 );
+				setTitle( '' );
+			}
+		} );
+	};
+
+	return (
+		<InspectorControls>
+			{ siteSlug && (
+				<ExternalLink
+					href={ `https://wordpress.com/earn/payments/${ siteSlug }` }
+					className={ 'product-management-control-inspector__link-to-earn' }
+				>
+					{ __( 'Manage your subscriptions.', 'jetpack' ) }
+				</ExternalLink>
+			) }
+			<PanelBody
+				title={ __( 'Add a new subscription', 'jetpack' ) }
+				initialOpen={ true }
+				className={ 'product-management-control-inspector__add-plan' }
+			>
+				{ apiState === API_STATE_REQUESTING && (
+					<Placeholder
+						icon={ lock }
+						label={ __( 'Premium Content', 'jetpack' ) }
+						instructions={ __( 'Saving plan…', 'jetpack' ) }
+					>
+						<Spinner />
+					</Placeholder>
+				) }
+				{ apiState === API_STATE_NOT_REQUESTING && (
+					<>
+						<PanelRow className="product-management-control-inspector__product-title">
+							<TextControl
+								id="new-product-title"
+								label={ __( 'Name', 'jetpack' ) }
+								onChange={ value => setTitle( value ) }
+								value={ title }
+							/>
+						</PanelRow>
+						<PanelRow className="product-management-control-inspector__product-price">
+							<SelectControl
+								label={ __( 'Currency', 'jetpack' ) }
+								onChange={ value => setCurrency( value ) }
+								options={ CURRENCY_OPTIONS }
+								value={ currency }
+							/>
+							<TextControl
+								label={ __( 'Price', 'jetpack' ) }
+								onChange={ value => setPrice( value ) }
+								type="number"
+								value={ price }
+							/>
+						</PanelRow>
+						<PanelRow className="plan-interval">
+							<SelectControl
+								label={ __( 'Interval', 'jetpack' ) }
+								onChange={ value => setInterval( value ) }
+								options={ intervalOptions }
+								value={ interval }
+							/>
+						</PanelRow>
+						<PanelRow>
+							<Button onClick={ handleSubmit } variant="secondary">
+								{ __( 'Add subscription', 'jetpack' ) }
+							</Button>
+						</PanelRow>
```

Comment:
In the Payments block, we are showing a link to the fees associated with our service. I think it would be a good idea to add it inside the controls so that we have a heterogeneous way of showing this information and therefore we wouldn't need to implement this link in each individual block that uses these controls.

The end result would look like this:
<img width="282" alt="Captura de pantalla 2022-03-19 a las 10 10 01" src="https://user-images.githubusercontent.com/1989914/159115246-d8169ec5-eb38-4404-a4e4-fe6ee096804c.png">
These would be the changes needed:
```suggestion
						<PanelRow>
							<Button onClick={ handleSubmit } variant="secondary">
								{ __( 'Add subscription', 'jetpack' ) }
							</Button>
						</PanelRow>
						<PanelRow>
							<ExternalLink href="https://wordpress.com/support/wordpress-editor/blocks/payments/#related-fees">
								{ __( 'Read more about Payments and related fees.', 'jetpack' ) }
							</ExternalLink>
						</PanelRow>
```
WDYT?

Reply:
Added to our backlog.